packages/react-examples/src/react-charting/GaugeChart/GaugeChartPage.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+
+import {
+  ComponentPage,
+  ExampleCard,
+  IComponentDemoPageProps,
+  PropertiesTableSet,
+} from '@fluentui/react-docsite-components';
+
+import { GaugeChartBasicExample } from './GaugeChart.Basic.Example';
+import { GaugeChartVariantExample } from './GaugeChart.Variant.Example';
+
+const GaugeChartBasicExampleCode =
+  require('!raw-loader?esModule=false!@fluentui/react-examples/src/react-charting/GaugeChart/GaugeChart.Basic.Example.tsx') as string;
+const GaugeChartVariantExampleCode =
+  require('!raw-loader?esModule=false!@fluentui/react-examples/src/react-charting/GaugeChart/GaugeChart.Variant.Example.tsx') as string;
+
+export class GaugeChartPage extends React.Component<IComponentDemoPageProps, {}> {
+  public render(): JSX.Element {
+    return (
+      <ComponentPage
+        title="Gauge Chart"
+        componentName="GaugeChartExample"
+        exampleCards={
+          <div>
+            <ExampleCard title="GaugeChart basic" code={GaugeChartBasicExampleCode}>
+              <GaugeChartBasicExample />
+            </ExampleCard>
+            <ExampleCard title="GaugeChart variant" code={GaugeChartVariantExampleCode}>
+              <GaugeChartVariantExample />
+            </ExampleCard>
+          </div>
+        }
+        propertiesTables={
```

Comment:
Add a basic description of the gauge chart and an image representation of different components of the gauge chart - like title, description, subtext etc

Reply:
done